meresco/xsd/nod-person-1-1.xsd

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- 
+<!--
 ****************************************************************
-*                            PERSON 1.1                                    
-*                                                                               
-*                            October 24, 2017                 
+*                            PERSON 1.1
+*
+*                            October 24, 2017

```

Comment:
@rvanheest Mag er een revisie datum bij? Dat is handig voor de (handmatige) boekhouding.